components/list/list.css

```diff
@@ -0,0 +1,52 @@
+@import url('/index.css');
+
+
+.list-container {
+    width: 28rem;
+    padding: 2rem;
+    border: 2px solid var(--gray-color);
+    border-radius: 5px;
+    background-color: var(--bg-color);
+}
+
+.simple-list{
+    width: 15rem;
+    padding: 1rem;
+    background-color: var(--white-color);
+    box-shadow: 2px 2px 5px var(--gray-color);
+}
+
+
+.list-heading {
+    padding: 0.5rem;
+}
+
+.list-parent {
+    text-decoration: none;
+    list-style: none;
+}
+
+.list-child {
+    padding: 0.5rem;
+    cursor: pointer;
+    border-radius: 5px;
+    display: flex;
+    align-items: center;
+}
+
+.list-child:hover {
+    background-color: rgb(233, 225, 225);
```

Comment:
use var for color for consistency as you used above

Reply:
done